### PR TITLE
fix title in legend table

### LINKED ISSUE #640: Legend table in PDF
The table of the legend should contain now: "Anteil" (instead of "Fläche") and "Anteil in %" instead of "Anteil".

### PR DESCRIPTION
**Pull request by vvmruder**
Reagarding to new specification... ;)
fixes #640

### CODE REVIEW

**Comment by vvmruder** [comment]
@voisardf & @lopo977 may I ask you for translation in the properties files of extract? :)

File: print/print-apps/oereb/Report_fr.properties [hunk 1]
```diff
@@ -29,6 +29,5 @@ CertificationLabel = Certification
 StampDateSignature = (Tampon)                                                                                   (Date)                                                                         (Signature)
 NoneGiven = -
 Type = Type
-AreaShare = Surface
```
Comment (vvmruder) [inline print/print-apps/oereb/Report_fr.properties]: @voisardf can you add translation here?

File: print/print-apps/oereb/Report_it.properties [hunk 2]
```diff
@@ -29,6 +29,5 @@ CertificationLabel = Certificazione
 StampDateSignature = (Timbro)                                                                                   (Data)                                                                          (Firma)
 NoneGiven = -
 Type = Tipo
-AreaShare = Superficie
```
Comment (vvmruder) [inline print/print-apps/oereb/Report_it.properties]: @lopo977 can you add translation here?

Comment (vvmruder) [reply print/print-apps/oereb/Report_it.properties]: Grazie! :+1:

**Comment by voisardf** [comment]
hmm, I'm not sure if I didn't push some other commits that shouldn't have come along... But changed files just indicate what we want to... 😕 ❓

**Comment by vvmruder** [comment]
No it looks good. I will merge this. :) Thanks for participation.

**Pull request merged**